CMakeLists.txt

```diff
@@ -1,21 +1,77 @@
 cmake_minimum_required(VERSION 3.19)
 cmake_policy(VERSION 3.19)
 
-# Initialize Mbed OS build system. 
-# Note: This block must be before the include of app.cmake
-set(MBED_APP_JSON_PATH mbed_app.json)
-set(CUSTOM_TARGETS_JSON_PATH custom_targets.json) # This activates the custom targets in custom_targets.json
+#### Initialize Mbed OS build system. ####
+######################################################################################################
+### Block of including .json5 files. Files of this block must be included before the app.cmake
 
+#[[ Set path of mbed_app.json (necessary everytime if mbed_app.json5 is in project) ]]
+set(MBED_APP_JSON_PATH mbed_app.json5)
+
+###--------------------------------------------------------------------------------------------------- 
+#[[ This part is intended to custom targets5 and activates the custom_targets.json5 
```

Comment:
I'm afraid I don't really understand what this comment is saying, could you rephrase?

Reply:
The plan was create a one file which will contain all basic informations for custom target project but also for regular project (could be also part of blinky example). So this should be just a mark of a sub-block under the including block related only to the custom target project settings.

Does the new description below look better?

> This part is dedicated for custom targets only! Settings below activate targets from custom_targets.json5 
and upload method config, otherwise functions below should be commented.